Tchap/Constants/DesignValues.m

```diff
@@ -122,7 +122,7 @@ - (void)userInterfaceThemeDidChange
 //    else
     {
         // Set light theme colors by default.
-        kVariant1PrimaryBgColor = kColorDarkBlue;
+        kVariant1PrimaryBgColor = [UIColor whiteColor];
```

Comment:
We have presently 2 variants: one based on DarkBlue and the other one on White.
The main difference between these 2 variants is the barTintColor of the navigationBar which depends on this color.
So kVariant1PrimaryBgColor was initially defined to set the navigationBar.barTintColor.
According to your PR, you are using it to defined the main view background color.

I suggest adding a new definition: kVariant1NavigationBgColor, or kVariant1DefaultBgColor